dvc/output.py

```diff
@@ -363,6 +369,15 @@ def __init__(
             name=self.hash_name,
             value=getattr(self.meta, self.hash_name, None),
         )
+        if self.hash_info and self.hash_info.isdir:
+            self.meta.isdir = True
+            if files:
+                tree = Tree.from_list(files, hash_name=self.hash_name)
+                tree.digest()
+                tree.meta = self.meta
```

Comment:
Not sure I understand why we are setting `tree.meta`. meta is not part of object.

Reply:
Doesn't seem to be used either. So i suppose it is just a typo.